Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
Loading gems from rubygems via SSL does not work for me. But sure, I can change that.